src/java.base/share/classes/java/security/SecureRandom.java

```diff
@@ -154,6 +160,8 @@ public class SecureRandom extends java.util.Random {
     private static final boolean skipDebug =
         Debug.isOn("engine=") && !Debug.isOn("securerandom");
 
+    private static boolean isFips = FIPSConfigurator.enableFips();
```

Comment:
Please remove this - it's only used once - use `FIPSConfigurator.enableFips()` directly instead.